galter_subjects_utils/reader.py

```diff
@@ -22,6 +23,14 @@ def read_jsonl(filepath):
             yield json.loads(line)
 
 
+def read_csv(filepath, reader_kwargs=None):
+    """KISS csv reader."""
+    reader_kwargs = reader_kwargs or {}
+    with open(filepath) as f:
+        reader = csv.DictReader(f, **reader_kwargs)
+        yield from reader
```

Comment:
Just adding for now? I don't see it being used, yet.

Reply:
Right, it's not used yet, but will be in the next one and it's small enough to include.